australian-guide-to-legal-citation.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
-    <title>Australian Guide to Legal Citation</title>
+    <title>Australian Guide to Legal Citation, 4th Edition</title>
```

Comment:
@adam3smith, minor nitpick, but we don't use a comma in any of our edition-versioned styles: https://www.zotero.org/styles?q=%20edition.